packages/patternfly-4/scripts/getPatternFlyCss.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+const request = require('request');
+
+const packageJson = require('../package.json');
+// get the specific version of patternfly as used by the react examples
+const pfVersion = packageJson.dependencies['@patternfly/src-patternfly-next'].split('#v')[1];
```

Comment:
I'm about to break this with submodules. I don't mind fixing it, just tell me what exactly it's doing.

Reply:
actually i think i'll take another look at this. basically i need to load one set of patternfly css for the core examples, and one set for the react examples, since they may not be in sync

Reply:
We fixed this making the version of @patternfly/patternfly in _repos/react-core exactly the version in _repos/core.